File: program/program/convert-feet-to-inches/convert_feet_to_inches.hs
module Main where

feetToInches :: Int -> Int
feetToInches i = i * 12

main :: IO ()
main = do
  -- Read a value from the user
  putStrLn "Enter a value in feet:"
  feet <- readLn
  -- Convert the value to inches and print the result
  let inches = feetToInches feet
  putStrLn $ show inches ++ " inches"

File: program/program/convert-feet-to-inches/ConvertFeetToInches.c
#include <stdio.h>

int main(void) {
  double feet;  
  double inches; 


  printf("Enter the number of feet: ");
  scanf("%lf", &feet);  

 
  inches = feet * 12; 

 
  printf("%.2lf feet is equal to %.2lf inches.\n", feet, inches);

  return 0;
}